internal/api/api.go

```diff
@@ -43,7 +43,14 @@ func Start(cfg *config.Config) error {
 	// Use metrics middleware without exposing path in main app router
 	metrics.UseWithoutExposingEndpoint(router)
 
-	// Custom metric
+	// Custom metrics
+	failureMetric := &ginmetrics.Metric{
+		// This is a Gauge because input-output-hk's is a gauge
```

Comment:
It makes sense to keep it inline with iohk, but it does seem like the wrong metric type?  While I _would_ expect the **rate** of errors to go up/down overtime I would think the failure count total should only ever be increasing?  With this being a gauge you will not be able to accurately track errors over time.

Not blocking because of first sentence.